maths/Tkinter_Calculator.py

```diff
@@ -0,0 +1,93 @@
+# calculator with some exception handling
+#Learn about Tkinter library form https://docs.python.org/3/library/tkinter.html
+
+
+import tkinter as tk 
+from tkinter import messagebox
+
+root = tk.Tk()
+root.title("Calculator")
+
+label = tk.Label(root, text="Enter first number", pady=(10))
+label.pack()
+
+first_number_entry = tk.Entry(root)
+first_number_entry.pack()
+
+label2 = tk.Label(root, text="Enter second number")
+label2.pack()
+
+second_number_entry = tk.Entry(root)
+second_number_entry.pack()
+
+operations = tk.Label(root, text="Operations")
+operations.pack()
```

Comment:
Please indent lines 8-24 under a function or a `if __name__ == "__main__":` so that our automated testing does not try to run it?

Reply:
Done! 
Thanks